scheduler/utilization_based_host_allocator.go

```diff
@@ -155,8 +156,13 @@ func evalHostUtilization(ctx context.Context, d distro.Distro, taskGroupData Tas
 	if d.HostAllocatorSettings.RoundingRule == evergreen.HostAllocatorRoundUp {
 		roundDown = false
 	}
+	numQOSTasks := numLongTasks
```

Comment:
what does QOS stand for?

Reply:
Quality Of Service

Reply:
I think this might be a confusing variable name; even with "numLongTasks", the calcNewHostsNeeded calls this variable "numHostsAlreadyNeeded" so there's maybe a disconnect between what the variable is named and what it's for. I would suggest "tasksNeedingHosts"? But this isn't a blocking comment.